.github/workflows/ci.yml

```diff
@@ -20,8 +20,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - uses: ./.github/actions/build
+      - name: Build and Push Docker Image
+        uses: ./.github/actions/build
         with:
           DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
           DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
-          TEMURIN_TAG: ${{ matrix.temurin_tag }}
\ No newline at end of file
+          TEMURIN_TAG: ${{ matrix.temurin_tag }}
+      - name: Active Snyk Project
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+        run: ./scripts/activate-snyk-project.sh ${{ secrets.SNYK_TOKEN }} java ${{ matrix.temurin_tag }} $(git rev-parse --short "$GITHUB_SHA")
```

Comment:
${{ secrets.SNYK_TOKEN }} is currently my personal API token. Could we update this to be a process account's token instead? Is there such a concept on Snyk?

Reply:
I think we could just create a 'user' that's for account updating if there isn't